versions/v6/readme.markdown

```diff
@@ -0,0 +1,7 @@
+# Pegasus Version 6 (v6) | Current Version
```

Comment:
Can we also update the v5 readme to note it as deprecated?  (ex: https://github.com/UCF/Pegasus-Theme/blob/master/versions/v4/readme.markdown)

Reply:
Done: https://github.com/UCF/Pegasus-Theme/pull/53/commits/08f763229a59dd0611feebf47521c0c357cf7666